### PR TITLE
MPU9250 experimental driver, largely inspired by https://github.com/k…

### PR DESCRIPTION
**Pull request by jdevelop**
…riswiner/MPU9250/

Provides some basic functionality for initialization, self-test,
calibrarion and reading the gyroscope and accelerometer data.

Uses SPI interface, I2C is yet to be implemented via the separate
transport.

Signed-off-by: Eugene Dzhurinsky <jdevelop@gmail.com>

### CODE REVIEW

**Comment by maruel** [comment]
gohci

File: experimental/devices/mpu9250/mpu9250.go [hunk 1]
```diff
@@ -0,0 +1,1936 @@
...
+func negateBool(src bool, err error) (bool, error) {
+	if err != nil {
+		return src, err
+	} else {
+		return !src, err
+	}
+}
+
+// AccelerometerXIsEnabled Get the X accelerometer status.
+func (m *
```
Comment (maruel) [inline experimental/devices/mpu9250/mpu9250.go]: This breaks `go vet -all -unsafeptr=false ./...` :(
https://gist.github.com/gohci-bot/dc3dc54e78a61be01f273a553e3719d2#file-cmd3-failed-in-4-487s
```
experimental\devices\mpu9250\mpu9250.go:1885: method WriteByte(address, value byte) error should have signature WriteByte(byte) error
```

**Comment by jdevelop** [comment]
gohci

File: experimental/devices/mpu9250/mpu9250.go (hunk 1 above)
Comment (jdevelop) [reply experimental/devices/mpu9250/mpu9250.go]: Sorry, I messed up with the branches, pushed the correct one. `go vet -all` passed just fine.

**Comment by maruel** [comment]
gohci

**Comment by maruel** [comment]
Sorry you need "super powers" (which I can grant you later).

**Pull request merged**